dist/index.js

```diff
@@ -89,17 +53,15 @@ function getLocation(v) {
 }
 
 function textMessage(v) {
+  let path = getLocation(v);
   var scheme = sourceScheme();
-  var prefix;
+  let prefix = `The path ${path} reports ${v.artifact.name} at version ${v.artifact.version} `;
+
   if (["dir", "tar"].includes(scheme)) {
-    prefix =
-      "The path " +
-      getLocation(v) +
-      " would result in an installed vulnerability: ";
+    return prefix + ` which would result in a vulnerable (${v.artifact.type}) package installed`;
   } else {
-    prefix = "The container image contains software with a vulnerability: ";
+    return prefix + `which is a vulnerable (${v.artifact.type}) package installed in the container`;
```

Comment:
nit: Usually you don't mix concatenation and templates (e.g. in this case you might use only templates)

Reply:
And this one too!